src/components/molecules/Dropdown/Dropdown.js

```diff
@@ -68,11 +71,20 @@ export default class Dropdown extends HTMLElement {
         this.parentElement
           .querySelector('cod-dropdown-menu')
           .setAttribute('data-show', 'false');
+        this.parentElement
+          .querySelector('cod-dropdown-menu') 
+          .setAttribute('aria-hidden', 'true');           
       } else {
         this.setAttribute('aria-expanded', 'true');
         this.parentElement
-          .querySelector('cod-dropdown-menu')
+          .querySelector('cod-dropdown-menu') 
           .setAttribute('data-show', 'true');
+        this.parentElement
+          .querySelector('cod-dropdown-menu') 
+          .setAttribute('aria-labelledby', '#button-id');
```

Comment:
We'll have to get the button ID from the DOM. Like this:


```js
const buttonID = this.parentElement
  .querySelector('cod-button')
  .getAttribute('id');
```

Then we can replace `'#button-id'` with `buttonID`.